python/paddle/tensor/manipulation.py

```diff
@@ -104,7 +104,7 @@ def flip(input, dims, name=None):
     return out
 
 
-def roll(input, shifts, dims=None):
+def roll(input, shifts, axis=None, name=None):
     """
```

Comment:
it should be : paddle.tensor.roll(x, shifts, axes=None, name=None)
according to the latest argument convention.

Reply:
ok